projects/plugins/jetpack/_inc/lib/admin-pages/class.jetpack-react-page.php

```diff
@@ -262,7 +262,7 @@ public function page_admin_scripts() {
 		$is_offline_mode     = $status->is_offline_mode();
 		$site_suffix         = $status->get_site_suffix();
 		$script_deps_path    = JETPACK__PLUGIN_DIR . '_inc/build/admin.asset.php';
-		$script_dependencies = array( 'wp-polyfill' );
+		$script_dependencies = array( 'jquery', 'wp-polyfill' );
```

Comment:
This is going to be overwritten by line 269.

I note that [the file read there](https://github.com/Automattic/jetpack-production/blob/8304ae4f6140fced2190d291a5e7defd1f38d71f/_inc/build/admin.asset.php) doesn't mention jquery either. The solution to that should be to fix the relevant components to import jquery instead of assuming it exists as a global so the dependency extraction plugin can find it, don't just munge it in the PHP code here.

(IIRC the only reason this one isn't already using `Assets::register_script()` is because [the css is being loaded entirely separately](https://github.com/Automattic/jetpack-production/blob/8304ae4f6140fced2190d291a5e7defd1f38d71f/class.jetpack.php#L3745-L3750) and the JS is loaded in [multiple](https://github.com/Automattic/jetpack-production/blob/8304ae4f6140fced2190d291a5e7defd1f38d71f/_inc/lib/admin-pages/class.jetpack-react-page.php#L273-L279) [places](https://github.com/Automattic/jetpack-production/blob/8304ae4f6140fced2190d291a5e7defd1f38d71f/class.jetpack-connection-banner.php#L252)).

Reply:
Thank you, @anomiex, I'd missed that codepath! 👍

I added the `import` to the component in question (plus some others) in 6ee3ac75d3cdd2e9571b10514c88bab3d552f6a8.